applications/games_menu/modules/menu.c

```diff
@@ -1550,16 +1550,28 @@ void ParseMenuConfigToPresentation()
 	{
 		menu_data.save_preset = true;
 	}
+	else
+	{
+		menu_data.save_preset = false;
+	}
 
 	if (menu_data.app_config.cover_background)
 	{
 		menu_data.cover_background = true;
 	}
+	else
+	{
+		menu_data.cover_background = false;
+	}
 
 	if (menu_data.app_config.change_page_with_pad)
 	{
 		menu_data.change_page_with_pad = true;
```

Comment:
Why don't you just do it like this?
`menu_data.change_page_with_pad = menu_data.app_config.change_page_with_pad;`

Reply:
Is bool fully compatible with int?

Reply:
> Is bool fully compatible with int?

Yeah I think the compiler will understand you correctly.
Or at least you can this:
`menu_data.change_page_with_pad = !!menu_data.app_config.change_page_with_pad;`

Reply:
This way I have used it many times.
menu_data.change_page_with_pad = menu_data.app_config.change_page_with_pad == 0 ? false : true;